babycare_partner_street_number/__openerp__.py

```diff
@@ -0,0 +1,14 @@
+{
+    "name": "Babycare Extension: Street name and number",
+    "summary": "Introduces new field for street number addition.",
+    "version": "8.0.0.1.0",
+    "author": "Wytze Jan Riedstra,Therp BV,Odoo Community Association (OCA)",
+    "category": 'Tools',
+    "depends": [
+        'partner_street_number',
```

Comment:
You need to depend on wk_base_partner_patch to ensure that this module's _display_address works on that module's override of the same method.

Reply:
Fixed.